Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/calculation/DccWalletInfoCalculation.kt

```diff
@@ -16,38 +15,33 @@ import de.rki.coronawarnapp.covidcertificate.common.certificate.CwaCovidCertific
 import de.rki.coronawarnapp.covidcertificate.validation.core.rule.DccValidationRule
 import de.rki.coronawarnapp.util.serialization.BaseGson
 import de.rki.coronawarnapp.util.serialization.BaseJackson
-import de.rki.jfn.JsonFunctions
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
 import javax.inject.Inject
 
 class DccWalletInfoCalculation @Inject constructor(
     @BaseJackson private val mapper: ObjectMapper,
-    @BaseGson private val gson: Gson
+    @BaseGson private val gson: Gson,
+    private val jsonFunctionsWrapper: JsonFunctionsWrapper,
 ) {
 
-    private lateinit var jsonFunctions: JsonFunctions
     private var boosterRulesNode: JsonNode = NullNode.instance
 
-    fun init(
-        cclConfiguration: CCLConfiguration,
-        boosterRules: List<DccValidationRule>
-    ) {
-        jsonFunctions = JsonFunctions()
+    fun init(boosterRules: List<DccValidationRule>) {
         boosterRulesNode = gson.toJson(boosterRules).toJsonNode()
-        cclConfiguration.logic.jfnDescriptors.forEach {
-            jsonFunctions.registerFunction(it.name, it.definition.toJsonNode())
-        }
     }
 
     fun getDccWalletInfo(
```

Comment:
Maybe make this function a `suspend` function so that we don't need to call `runBlocking{}`

Reply:
if it is a suspend function, the compiler complains about mapper.treeToValue being blocking

Reply:
Is it a warning? I don't see anything on my machine...